build.gradle

```diff
@@ -35,45 +46,76 @@ dependencyManagement {
 
 configurations {
     documentation
+
+    configureEach {
+        resolutionStrategy.dependencySubstitution {
+            substitute module("org.codehaus.groovy:groovy-all") using module("org.apache.groovy:groovy:$groovyVersion")
+        }
```

Comment:
Is this necessary? What's pulling in `groovy-all`?

Reply:
Good catch, it was no longer necessary and has been removed entirely.